app/search.py

```diff
@@ -15,12 +15,32 @@ def open_browser(urls):
 
 def search_my_term(term, count):
     return [url for url in search(term, stop=count)]
+def search_rem(term,s,count):
```

Comment:
`search_rem` and `search_my_term` looks so same, why don't we make it one function with start and stop?